app/models/add_on.rb

```diff
@@ -12,11 +14,14 @@ class AddOn < ApplicationRecord
   monetize :amount_cents
 
   validates :name, presence: true
-  validates :code, uniqueness: { scope: :organization_id, allow_nil: false }
+  validates :code,
```

Comment:
Maybe we can add an example as we did on the others, no?
https://github.com/getlago/lago-api/blob/main/spec/services/plans/create_service_spec.rb#L96

Reply:
Yeah indeed, good catch!